internal/filesystem/filesystem.go

```diff
@@ -104,7 +104,7 @@ func OpenFile(fsys fs.FS, name string, flag int, perm os.FileMode) (fs.File, err
 	if flag == os.O_RDONLY {
 		return fsys.Open(name)
 	}
-	return nil, fmt.Errorf("open %s: operation not supported", name)
+	return nil, fmt.Errorf(`open "%s": operation not supported`, name)
```

Comment:
This extension is taken directly from rsc and rob's proposal.
The format can stay this way for now, but when we tackle error values more broadly, we should add and wrap some kind of pkg defined error.
Something like `ExtensionNotImplementedError` or whatever, so that callers (mainly testing code) can better distinguish this from other generic `Open` errors.